templates/enter2fa.html

```diff
@@ -55,6 +56,7 @@ <h2>Confirm you're you!</h2>
                             <input class='validate otp' type='number' id='otp' max="999999" minlength="6" maxlength="6" autofocus/>
                             <label for='otp'>Use the entry starting with "<span name="data-connection-name"></span>"</label>
                             <span class="helper-text" data-error="Must be 6 digits" data-success="right"></span>
+							<button type="button" class="waves-effect waves-light btn" id="otp-button">Submit</button>
```

Comment:
Same question as above 😄

Reply:
Yes, auto submit on keypress does not work in a mobile browser.